student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def classroom(classes)
+    @classroom = classes
+    classes.students.push(self) unless classes.students.include(self)
```

Comment:
When setting the classroom for a student it also adds it to the classrooms students ✔